test/unit/zeromq-node-gyp/input.js

```diff
@@ -0,0 +1 @@
+require('zeromq');
```

Comment:
A unit test is great! Let's also add an integration test. Basically copy this same file to `test/integration/zeromq.js`

Reply:
Done.